uninstall.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Fired when the plugin is uninstalled.
+ */
+ 
+/* If uninstall not called from WordPress, then exit. */
+if ( ! defined( 'WP_UNINSTALL_PLUGIN' ) ) {
+	wp_die( sprintf( __( '%s should only be called when uninstalling the plugin.', 'wdp-plugin' ), '<code>' . __FILE__ . '</code>' ) );
+}
+
+
+/* Deleting FFPW widget option row in database */
+delete_option('widget_FFPW_Flexible_Featured_Post');
+
+
+/* Deleting post meta data added by plugin */
+function delete_ffpw_meta() {
+   
+    global $wpdb;
```

Comment:
@coder-karen is there a need to globalise this variable as it doesn't look as though it is used?